lib/go/payloads/BUILD

```diff
@@ -1,15 +1,18 @@
 load("@com_github_airyhq_bazel_tools//lint:buildifier.bzl", "check_pkg")
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
-# gazelle:prefix github.com/airyhq/airy/lib/go/httpclient/payloads
+# gazelle:prefix github.com/airyhq/airy/lib/go/payloads
```

Comment:
Moved the `payloads` outside of the `httpclient` package, because they are also used by the controller.